float64[] posture
float64[] posture_prob
float64[] centroid_x
float64[] centroid_y
float64[] distance
float64[] orientation
float64[] image_width
float64[] camera_id
